orderedmap.go

```diff
@@ -95,20 +130,45 @@ func (om *OrderedMap[K, V]) Len() int {
 	return len(om.pairs)
 }
 
+// Size is an alias for Len.
+func (om *OrderedMap[K, V]) Size() int {
+	return len(om.pairs)
```

Comment:
That is an unnecessary optimization to not just call Len. Replacing with Len, then running this below shows that the go compiler is smart enough to inline:
```
~/gocode/go-ordered-map $ cat dummy/main.go 
package main

import (
	"fmt"
	"github.com/wk8/go-ordered-map/v2"
)

func main() {
	om := orderedmap.New[string, string]()
	fmt.Println(om.Size())
}
~/gocode/go-ordered-map $ go build -gcflags -m -o $(mktemp) github.com/wk8/go-ordered-map/v2/dummy |& grep inline | grep 134
/Users/wk/gocode/go-ordered-map/orderedmap.go:134:29: can inline orderedmap.(*OrderedMap[go.shape.string_0,go.shape.string_1]).Size
```